### DISCUSSION TITLE
References to a commit's SHA hash can't are automatically converted into shortened links

### DISCUSSION THREAD

**Discussion by ipcjs** [community/community#40821] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Some time ago it was no problem, but recently it suddenly could not be converted automatically.

![image](https://user-images.githubusercontent.com/4396864/205631190-2f15ab57-dfd0-48bf-850d-939cd6a63449.png)

**Comment by ossanna16** [comment]
Hi there @ipcjs and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
